### PR TITLE
Update blog post to include change in paths for kernel version 4.14

### PR DESCRIPTION
**Pull request by taniaduggal**
Fix #32011

### CODE REVIEW

**Comment by Sea-n** [comment]
Please don't take steal others' hard work to be your PR, it's very unethical.
- #32011

/hold

**Comment by sftim** [comment]
PRs to fix issues are welcome.

This doesn't look to me like “stealing” - open source is a thing. However, @taniaduggal, you could have been more clear that you were building on existing work.

@taniaduggal it is best to see if there is already an open PR and, if so, to try to liaise with the person who already opened that one.

/hold cancel

**Comment by taniaduggal** [comment]
Hey @sftim , that user is inactive from long time. That's why , i raised a PR since I believed the changes were crucial. 
Please  review at my pr.
/assign @sftim

**Comment by taniaduggal** [comment]
Hey @sftim , that user is inactive from long time . That's why, i raised a PR 
since I believed changes were crucial .
Please review at my PR.

**Comment by taniaduggal** [comment]
> Hey @sftim , that user is inactive from long time. That's why , i raised a PR since I believed the changes were crucial. Please review at my pr. /assign @sftim

**Comment by taniaduggal** [comment]
> Hey @sftim , that user is inactive from long time. That's why , i raised a PR since I believed the changes were crucial. Please review at my pr.

**Comment by taniaduggal** [comment]
> Hey @sftim , that user is inactive from long time. That's why , i raised a PR since I believed the changes were crucial. Please review at my pr.

File: content/en/blog/_posts/2019-03-29-kube-proxy-subtleties-debugging-an-intermittent-connection-resets.md [hunk 1]
```diff
@@ -139,15 +139,20 @@ Once we understand the root cause, the fix is not hard. There are at least 2
...
 
 The [fix](https://github.com/kubernetes/kubernetes/pull/74840) is available in v1.15+.
 However, for the users that are affected by this bug, there is a way to mitigate the
 problem by applying the following rule in your cluster.
 
+_Update_: With worker nodes on kernel version 4.14 and above, the path mentioned in the script to set the sysctl might be different.
+The recommendation is to connect to the worker node through SSH and run the following the command.
+`echo 1 > /proc/sys/net/ipv4/netfilter/ip_conntrack_tcp_be_liberal`.
+If the command returns `No such file or directory`, that means the netfilter directory likely uses the sysctl path`/proc/sys/net/netfilter/nf_conntrack_tcp_be_liberal`.
+You must update the command in the DaemonSet below to use the correct path.
```
Comment (sftim) [inline content/en/blog/_posts/2019-03-29-kube-proxy-subtleties-debugging-an-intermittent-connection-resets.md:154]: This seems to be the only / key change. @taniaduggal are you using an editor that automatically reformats Markdown?

We'd prefer to only see the relevant changes. To do that, you could copy this section and redo your PR from the _main_ branch. When you come to commit your work, use `git add -p` or the equivalent IDE tool to let you select which changes to commit.

**Review by sftim — CHANGES_REQUESTED** [review]
Thanks for opening this pull request @taniaduggal 

I'm afraid that as things stand we couldn't accept this change

- PRs that change blog articles more than 1 year old should include an explanation about why we should make the change (the default policy is not to amend articles over a year old)
- this PR as I see it has a bunch of unrelated changes, and I'm afraid we'd prefer not to accept those
- the author, Yonkun Gui, might have a view on this amendment. Please mention in your PR description whether you have / haven't tried to get in touch with the original author

**Comment by taniaduggal** [comment]
Hey @sftim .I made only these changes which you suggested . You once have to check the original file .
and also i am beginner to open source . Any suggesstions are welcome .

**Pull request closed without merging**